src/snowflake/snowpark/mock/_functions.py

```diff
@@ -308,6 +309,26 @@ def mock_to_date(
     )
 
 
+@patch("current_timestamp")
+def mock_current_timestamp():
+    return ColumnEmulator(
+        data=datetime.datetime.now(),
+        sf_type=ColumnType(TimestampType(TimestampTimeZone.LTZ), False),
+    )
+
+
+@patch("current_date")
+def mock_current_date():
+    now = datetime.datetime.now()
+    return ColumnEmulator(data=now.date(), sf_type=ColumnType(DateType(), False))
+
+
+@patch("current_time")
+def mock_current_time():
+    now = datetime.datetime.now()
+    return ColumnEmulator(data=now.time(), sf_type=ColumnType(TimeType(), False))
```

Comment:
do we need to take timezone into account, I'm wondering if is there any session parameter that could change the returning value?

Reply:
Timezone does affect the result, but I don't think we have a good mechanism for overriding session parameters in local testing yet and creating that mechanism is outside of the scope for this change.

Reply:
yeah, I also don't think it's an issue to be addressed now.
can you create a JIRA for this timezone session parameter and put it into our backlog epic?